tests/torch/nas/test_all_elasticity.py

```diff
@@ -397,7 +399,10 @@ def test_can_restore_from_state():
     _, training_ctrl = resume_compression_from_state(nncf_network, old_state, empty_nncf_config)
 
     new_state = training_ctrl.get_compression_state()
-    assert new_state == old_state
+    assert old_state == REF_COMPRESSION_STATE_FOR_TWO_CONV
+    REF_NEW_STATE = deepcopy(REF_COMPRESSION_STATE_FOR_TWO_CONV)
+    REF_NEW_STATE['builder_state']['progressive_shrinking'].update({'bn_adaptation_params': {}})
```

Comment:
why do you need this? why are `bn_adapt_params` not saved as before?
according to the code, it should be saved:
https://github.com/openvinotoolkit/nncf/pull/1329/files#diff-80b9e15adbbbec089ebbe4a96e83fac57b8434bc086d5404ce2cd084e896a7b1R110

Reply:
Thanks! I have reverted to the previous form of the test, and fixed how bn_params are handled.